src/Components/CreateImageWizard/ValidatedTextInput.tsx

```diff
@@ -30,6 +34,61 @@ interface HookValidatedTextInputPropTypes extends TextInputProps {
   warning?: string;
 }
 
+export const HookPasswordValidatedInput = ({
+  ariaLabel,
+  placeholder,
+  dataTestId,
+  value,
+  ouiaId,
+  stepValidation,
+  fieldName,
+  onChange,
+  warning = undefined,
+}: HookValidatedTextInputPropTypes) => {
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const togglePasswordVisibility = () => {
+    setIsPasswordVisible(!isPasswordVisible);
+  };
+
+  return (
+    <>
+      <InputGroup>
+        <InputGroupItem isFill>
+          <HookValidatedInput
+            isRequired
```

Comment:
Hmm, should this be marked as a required field? The user can either fill in the password or ssh key, right? So it's kinda required, but not really. I guess we can leave it now, but could maybe check how to indicate that password or ssh key need to be filled in.

Reply:
I will add the check if password or ssh key filled in the validation function